src/Sulu/Bundle/AdminBundle/Controller/AdminController.php

```diff
@@ -72,7 +74,7 @@ public function __construct(
         private array $locales,
         private array $translations,
         private string $fallbackLocale,
-        private string $collaborationInterval,
+        private int $collaborationInterval,
```

Comment:
Technically a break, because strict types = 0 allowed it to be a numeric string and still work.